packages/react-devtools-shared/src/backend/agent.js

```diff
@@ -141,6 +141,10 @@ type PersistedSelection = {
   path: Array<PathFrame>,
 };
 
+export const initialProfileRef = {
```

Comment:
Nit: We should probably have a getter/setter for `shouldProfileOnStart` or something instead of making this a ref for safety.

Reply:
Sounds good.